test/ibmq/test_basic_server_paths.py

```diff
@@ -86,6 +86,8 @@ def test_retrieve_jobs(self):
                 self.assertGreaterEqual(len(retrieved_jobs), 1)
                 retrieved_job_ids = {job.job_id() for job in retrieved_jobs}
                 self.assertIn(job_id, retrieved_job_ids)
+                # Cancel job so it doesn't consume more resources.
+                cancel_job(job, verify=True)
```

Comment:
Job cannot be cancelled if they're in certain states. So unless you are sure the job is in a cancellable state, don't use `verify=True`.